src/cpp/proxy/websocketoverhttp.cpp

```diff
@@ -78,7 +80,7 @@ class WebSocketOverHttp::DisconnectManager : public QObject
 	void addSocket(WebSocketOverHttp *sock)
 	{
 		sock->setParent(this);
-		connect(sock, &WebSocketOverHttp::disconnected, this, &DisconnectManager::sock_disconnected);
+		disconnectedConnection[sock] = sock->disconnected.connect(boost::bind(&DisconnectManager::sock_disconnected, this, sock));
```

Comment:
The connection needs to be removed from the map too, probably in `cleanupSocket`.

Reply:
wow! nice catch!